app/views/unauthenticatedError.scala.html

```diff
@@ -0,0 +1,16 @@
+@*
+* This is a placeholder to allow us to write unit tests which return an error page.
+* It will be replaced once we have a properly designed error page
+*@
+@(errorMessage: String)(implicit messages: Messages)
+
+@main(Messages("error.header")) {
+    @defining(play.core.PlayVersion.current) { version =>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <h1 class="govuk-heading-l">You must be logged in to see this page</h1>
+                <p class="govuk-body"> Please <a class="govuk-link" href="#">login</a> to continue.</p>
```

Comment:
I've updated the acceptance criteria for [TDR-583](https://national-archives.atlassian.net/browse/TDR-583) to remind us to make this link work.